test/models/search_facet_test.rb

```diff
@@ -69,4 +72,31 @@ def setup
 
     puts "#{File.basename(__FILE__)}: #{self.method_name} successful!"
   end
+
+  test 'should set data_type and is_array_based on create' do
+    puts "#{File.basename(__FILE__)}: #{self.method_name}"
+
+    mock = Minitest::Mock.new
+    mock.expect :query, @column_schema, [String]
+
+    SearchFacet.stub :big_query_dataset, mock do
+      age_facet = SearchFacet.create(
+          name: 'Organism Age',
+          identifier: 'organism_age',
```

Comment:
so it's less confusing, I'd use 'sex' as the example facet here instead of 'organism_age' here, since you're saying it's not ontology based.